### PR TITLE
Window: use more actions for keyboard shortcuts

### PR DESCRIPTION
**Pull request by danrabbit**
Use actions for a few more keyboard shortcuts. Saves us some confusing manually key handling and makes Gtk4 porting easier

### CODE REVIEW

File: src/Window.vala [hunk 1]
```diff
@@ -44,23 +44,33 @@ public class Audience.Window : Gtk.ApplicationWindow {
...
         { ACTION_FULLSCREEN, action_fullscreen },
         { ACTION_OPEN_FILE, action_open_file },
         { ACTION_QUIT, action_quit },
+        { ACTION_SEARCH, action_search },
+        { ACTION_UNDO, action_undo }
     };
 
     static construct {
+        action_accelerators[ACTION_BACK] = "<Alt>Left";
+        action_accelerators[ACTION_BACK] = "Back";
```
Comment (jeremypw) [inline src/Window.vala:67]: Do you mean "BackSpace"?  Or is there there another special "Back" key on some keyboards?

File: src/Window.vala [hunk 2]
```diff
@@ -350,27 +386,6 @@ public class Audience.Window : Gtk.ApplicationWindow {
...
                     break;
             }
-        } else if (deck.visible_child == welcome_page) {
-            if (match_keycode (Gdk.Key.p, keycode) || match_keycode (Gdk.Key.space, keycode)) {
-                resume_last_videos ();
-                return true;
-            } else if (ctrl_pressed && match_keycode (Gdk.Key.b, keycode)) {
-                show_library ();
-                return true;
-            }
```
Comment (jeremypw) [inline src/Window.vala:360]: You do not seem to have implemented any actions to replace these shortcuts.

File: src/Window.vala (hunk 1 above)
Comment (jeremypw) [inline src/Window.vala:67]: Did you mean "BackSpace"?

**Review by jeremypw — CHANGES_REQUESTED** [review]
See inline comments

File: src/Window.vala (hunk 1 above)
Comment (danrabbit) [reply src/Window.vala:67]: Yeah this is for the back media key which has precedent in Switchboard here: https://github.com/elementary/switchboard/blob/b25fcb65e32fe5462fa893abd14168db652c961c/src/Application.vala#L136

Comment (jeremypw) [reply src/Window.vala:67]: OK

File: src/Window.vala (hunk 2 above)
Comment (danrabbit) [reply src/Window.vala]: Oops, reverted!

**Review by jeremypw — APPROVED** [review]

**Pull request merged**